controller/TransparencyController.js

```diff
@@ -0,0 +1,50 @@
+const { Donation } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description get total donations
+ * @route GET /v1/transparency
+ * @access PUBLIC
+ */
+exports.getTotalAmount = catchAsyncError(async (req, res) => {
```

Comment:
Let's rename this function to `getTotalDonationsWithinTimeRange`, that makes this more descriptive than just saying `getTotalAmount`.